rpc/gnmi/gnmi-specification.md

```diff
@@ -1484,18 +1484,21 @@ that fewer messages are sent to the client. The advantage of such bundling is
 clearly to reduce the number of bytes on the wire (caused by message overhead);
 however, since only `Notification` messages contain the timestamp at which an
 event occurred, or a sample was taken, such bundling assigns a single timestamp
-for all bundled `Update` values. As such, it has the downside of negatively
-affecting the sample accuracy and freshness to the client, and as a result, on
-the client's ability to react to events on the target.
-
-Since it is not possible for the target to infer whether its clients are
-sensitive to the latency introduced by bundling, if a target implements
-optimizations such that multiple `Update` messages are bundled together,
-it MUST provide an ability to disable this functionality within the
-configuration of the gNMI service. Additionally, a target SHOULD provide means
-by which the operator can control the maximum number of updates that are to be
-bundled into a single message,  This configuration is expected to be implemented
-out-of-band to the gNMI protocol itself.
+for all bundled `Update` values. As such, bundling is primarily useful for
```

Comment:
I wonder whether we should have an additional sentence here that says something like. 

> Bundling of updates MUST NOT cause loss of fidelity in the provided timestamp.

to be extremely clear as to saying what the priority is?

Reply:
I attempted this in the subsequent paragraph...

Reply:
I agree - but I think a pithy statement of this nature makes the point more clear. It's just the TLDR that someone should leave with.